src/TreeView/__stories__/__snapshots__/TreeView.visual.storyshot

```diff
@@ -533,39 +533,39 @@ exports[`Storyshots Visual Avatar 1`] = `
       className="fddocs-container"
     >
       <span
```

Comment:
If you merge in master and rerun your storybook tests this bug should be resolved https://github.com/SAP/fundamental-react/pull/1092